BoxContentSDK/BoxContentSDK/Requests/BOXFileRequest.m

```diff
@@ -262,6 +262,23 @@ - (void)setRepresentationRequestOptions:(BOXRepresentationRequestOptions)represe
     self.representationsRequested = [[NSMutableOrderedSet alloc] initWithArray:arguments];
 }
 
+// Swift compatable va_list version of the method
+- (void)setRepresentationRequestOptions:(BOXRepresentationRequestOptions)representationOptions args:(va_list)argumentList
+{
+    NSMutableArray *arguments=[[NSMutableArray alloc]init];
+    BOXRepresentationRequestOptions eachObject;
+    if (representationOptions) {
+        [arguments addObject: [NSNumber numberWithUnsignedInteger:representationOptions]];
+        while ((eachObject = va_arg(argumentList, BOXRepresentationRequestOptions))) {
+            if ([NSNumber numberWithUnsignedInteger:eachObject] > 0) {
```

Comment:
This will always be `> 0`, because it's non-`nil`

Reply:
done